container/worker/run_openqa_worker.sh

```diff
@@ -6,4 +6,8 @@ if [[ -z $OPENQA_WORKER_INSTANCE ]]; then
 fi
 
 /root/qemu/kvm-mknod.sh
+
+groupmod -g "$(ls -lhn /dev/kvm | cut -d ' ' -f 4)" kvm
+usermod -a -G kvm _openqa-worker
```

Comment:
I assume this will fail unless the group "kvm" exists which I would not rely on as already stated multiple times: qemu is the most common but just *one* backend, not the only one that can be used

Reply:
It would make sense to check whether the group `kvm` and `/dev/kvm` both exist before running this code.

Reply:
Ok thanks

Reply:
done